### PR TITLE
Edit category

### PR DESCRIPTION
**Pull request by haonh**

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -95,3 +95,6 @@ en:
   create_word_failed_message: "Word content can't be blank"
   create_word_success_message: "Create word successful"
   destroy_word_success_message: "Destroy word successful"
+  update_category: "Update category"
+  update_successfull_message: "Update successful"
+
```
Comment (thiennv57) [inline config/locales/en.yml]: Add new line

File: app/controllers/admin/categories_controller.rb [hunk 2]
```diff
@@ -6,6 +6,18 @@ def index
...
 
+  def edit   
+  end
+
+  def update
+    if @category.update category_params
+      flash[:success] = t("update_successfull_message")
+      redirect_to admin_categories_path
+    else
+      render :edit
```
Comment (ttuan) [inline app/controllers/admin/categories_controller.rb]: thêm flash lúc update fail

**Comment by ThuBM** [comment]
OK

File: app/views/categories/_category.html.erb [hunk 3]
```diff
@@ -1,14 +1,13 @@
...
-      <% end %> 
-    </span>
-  </div>
+<div class="row">
+  <span class="col-md-10">
+    <div class="categoryName"><%= category.name %> </div>
+  </span>
+  <span>
+    <%= form_for Lesson.new do |f| %>
+      <%= f.hidden_field :category_id, value: category.id %>
```
Comment (chuanhtuan) [inline app/views/categories/_category.html.erb]: remove

**Pull request merged**